tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -461,6 +462,23 @@ describe(__filename, () => {
     expect(screen.getAllByText(errorString)).toHaveLength(3);
   });
 
+  it('shows Login Expired if api returns 401 auth expired', async () => {
+    signInUserAndRenderUserProfile();
+
+    createFailedErrorHandler({
```

Comment:
I know I wrote the test that this is based on, but I do have a suggestion. Just to prove that the API result is what is causing the page to be shown, perhaps we should add an assert before this to verify that we are on the User Profile page?

Reply:
I guess this will do? https://github.com/mozilla/addons-frontend/pull/11785/commits/12dd1bee84db2550e099e1c952f80da759544515  (I chose something that would only appear on your own profile page, once logged in)